src/editors/containers/ProblemEditor/components/EditProblemView/SettingsWidget/messages.js

```diff
@@ -181,7 +181,8 @@ export const messages = {
   },
   ConfirmSwitchMessage: {
     id: 'authoring.problemeditor.settings.switchtoadvancededitor.message',
-    defaultMessage: 'If you use the advanced editor, this problem will be converted to OLX and you will not be able to return to the simple editor.',
+    defaultMessage: `If you use the advanced editor, this problem will be converted to OLX and you will not be able to return to the simple editor.
+      Any changes made after the last Save action will be lost when converting to OLX.`,
```

Comment:
Do you intend for this to be on a new line? If yes, it is not rendering that way and you might have to enter a newline character.

Reply:
I did this for code readability but it should be a new line.